pages/[[...swingset]].jsx

```diff
@@ -44,12 +45,64 @@ function Index() {
   )
 }
 
-const { code, pre, CodeBlockConfig, CodeTabs } = codeMdxComponents({
+const { pre, CodeBlockConfig, CodeTabs } = codeMdxComponents({
   theme: 'light',
 })
 
+const mdxHeadings = {
```

Comment:
The `headings` and `table` work here are backfilling Swingset issues I created in https://github.com/hashicorp/react-components/pull/271 .